portfolio/src/main/webapp/script.js

```diff
@@ -262,3 +272,31 @@ function createLocationMarker(location, map) {
   });
   return marker;
 }
+
+function centerAthens(){
+  initCenteredTravelMap(athens)
+}
+
+function centerPrague(){
+  initCenteredTravelMap(prague)
+}
+
+function centerFlorence(){
+  initCenteredTravelMap(florence)
+}
+
+function centerLondon(){
+  initCenteredTravelMap(london)
+}
+
+function centerSalzburg(){
+  initCenteredTravelMap(salzburg)
+}
+
+function centerVatican(){
+  initCenteredTravelMap(vatican)
+}
+
+function centerBudapest(){
+  initCenteredTravelMap(budapest)
+}
```

Comment:
Nit: Can we omit the function declarations and call `initCenteredTravelMap` directly in the HTML? Having separate methods for each place seems unnecessary and not so convenient for maintenance as you'll travel a lot more in the future I hope :D

Reply:
thank you so much for the suggestions Kazi! :D